CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+### 3.17.0
+
+- Add constructor method that accepts zip and optionally modules. This constructor will be the official constructor in docxtemplater v4, and the methods : `loadZip`, `attachModule` and `compile` will no more be available. 
+
```

Comment:
Please add here : 

 You can migrate the following code :

Reply:
Aww, man! apologies for the stupid mistake.